gremlin/templates/gremlin-service-account.yaml

```
@@ -9,6 +9,28 @@ metadata:
     {{- with .Values.gremlin.serviceAccount.annotations }}
       {{- toYaml . | nindent 4 }}
     {{- end }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: gremlin-daemon-watcher
```

nit: let's call it something else to more differentiate it from the chao agent (which is doing more of what I'd consider "watching").

```suggestion
  name: gremlin-node-metadata-reader
```

---

sounds good to me, I'll also update line 33